autoload/pythonimports.vim

```diff
@@ -7,17 +7,24 @@ function! pythonimports#filename2module(filename)
   let pkg = fnamemodify(a:filename, ":p")
   let root = fnamemodify(pkg, ":h")
 
+  "normalize  paths
+  let _tmp = []
+  for path in g:pythonPaths
+    call add(_tmp, expand(path:p))
+  endfor
+  let g:pythonPaths = _tmp
+
   let found_dir = ""
   let found_path = ""
   while root != ""
-    if index(g:python_paths, root) != -1
+    if index(g:pythonPaths, root:p) != -1
```

Comment:
That does not look like valid syntax to me.

(`root` already holds an absolute path, see lines 7 and 8 in this file.)